java/src/main/java/ai/onnxruntime/OnnxSparseTensor.java

```diff
@@ -0,0 +1,876 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+ * Licensed under the MIT License.
+ */
+package ai.onnxruntime;
+
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.IntBuffer;
+import java.nio.LongBuffer;
+import java.util.Arrays;
+
+/**
+ * A Java object wrapping an OnnxSparseTensor.
+ *
+ * <p>Sparse tensors support a variety of formats, and the {@link #getValue} method returns a
+ * different static inner class representing each type.
+ */
+public final class OnnxSparseTensor extends OnnxTensorLike {
```

Comment:
>OnnxSparseTensor [](http://example.com/codeflow?start=19&length=16)

Just a remark. Onnx is a standard. Onnxruntime is an implemenation. Technically, we deal with a Onnxruntime implementation of tensors.

Reply:
The `Onnx...` classes map onto concepts in ONNX, and the `Ort...` classes map onto ONNX Runtime specific concepts. I admit the line is a bit fuzzy here because ONNX doesn't define sparse tensor formats, but it does define generic sparse tensors as a thing.